site/content/docs/11-deploying.md

```diff
@@ -61,4 +61,72 @@ When deploying to [Now][], you can pass the environment variable into Now itself
 now -e SAPPER_TIMESTAMP=$(date +%s%3N)
 ```
 
-[Now]: https://zeit.co/now
\ No newline at end of file
+[Now]: https://zeit.co/now
+
+
+## Deploying with Docker
+
+[Docker](https://docs.docker.com/install/) can be used to build an image and run it on your local system. Doing this locally allows testing the build process before building and deploying it on a cloud platform. The Docker image can be build by adding a `Dockerfile` into your project:
```

Comment:
"The Docker image can be build by adding a `Dockerfile` into your project" -> "A Docker image can be built by adding a `Dockerfile` to your project"